integration-tests/debugger/utils.js

```diff
@@ -53,14 +53,16 @@ function setup ({ env, testApp, testAppSource, dependencies, silent, stdioHandle
   }
 
   // Trigger the breakpoint once probe is successfully installed
-  function triggerBreakpoint (url) {
+  async function triggerBreakpoint (url) {
     let triggered = false
-    t.agent.on('debugger-diagnostics', ({ payload }) => {
-      payload.forEach((event) => {
-        if (!triggered && event.debugger.diagnostics.status === 'INSTALLED') {
-          triggered = true
-          t.axios.get(url)
-        }
+    return new Promise((resolve, reject) => {
+      t.agent.on('debugger-diagnostics', ({ payload }) => {
```

Comment:
Could this trigger multiple times? If yes, do we expect it to be in any of the payloads or always in the first?
I would normally switch to `once`.